.springBeans

```diff
@@ -1,15 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hey @rajadilipkolli - out of curiosity - why are you removing this particular file?
I'm not against removing it, but it's not related to the topic of this PR.